src/style.css

```diff
@@ -0,0 +1,35 @@
+body,
+div {
+  margin: 0 auto;
```

Comment:
why do you assign this to body?

Reply:
because there are default margins of 8px on body, 
and I need to set both vertical margins to zero:

> There should be no vertical spacing between the blocks and the edges of the page.

P.S. `npm test` can't catch error on bottom edge, if I set only `margin-top: 0;`

Reply:
you should write:
```
body {
  margin: 0;
}

div {
  margin: 0 auto;
}
```